docs/rules/no-child-traversal-in-connectedcallback.md

```diff
@@ -0,0 +1,54 @@
+# Disallows traversal of children in the `connectedCallback` method (no-dom-traversal-in-connectedcallback)
```

Comment:
name needs updating to `no-child-traversal-in-connectedcallback`

Reply:
Oops I thought I caught all these!